Sources/SourceKitLSP/Swift/CodeCompletion.swift

```diff
@@ -169,18 +169,33 @@ extension SwiftLanguageServer {
       // Map SourceKit's not_recommended field to LSP's deprecated
       let notRecommended = (value[self.keys.not_recommended] as Int?).map({ $0 != 0 })
 
-      let kind: sourcekitd_uid_t? = value[self.keys.kind]
+      let kind = (value[self.keys.kind] as sourcekitd_uid_t?)?.asCompletionItemKind(self.values) ?? .value
+      var usr: String? = nil
+      
+      switch kind {
+        // Don't fill in `usr` for things other than Swift symbols.
+        case .class, .struct, .constant, .variable, .enum, .enumMember, .constructor,
+             .field, .function, .interface, .method, .module, .operator, .property, .typeParameter:
```

Comment:
Which cases do we need this check for? Couldn’t we just check if the `associated_usrs` key is available in `value`?